core-plugins/widgets/File-batchsink.json

```diff
@@ -32,11 +32,14 @@
           "name": "format",
           "widget-attributes": {
             "values": [
-              "text",
```

Comment:
'text' was replaced with 'delimited', but they had the same behavior. File sink is new in this release, otherwise this would be a backward incompatible changed.

Using 'delimited' instead of 'text', because 'text' in the source means everything gets read into a single 'body' field so they're different.